src/data/nat/factorization.lean

```diff
@@ -297,4 +297,27 @@ begin
     exact h p _ pp (pow_factorization_dvd p _) },
 end
 
+lemma factorization_gcd (a b : ℕ) (ha_pos : a ≠ 0) (hb_pos : b ≠ 0) :
```

Comment:
For consistency with other lemmas, these should probably be implicit arguments:
```suggestion
lemma factorization_gcd {a b : ℕ} (ha_pos : a ≠ 0) (hb_pos : b ≠ 0) :
```
Other than that, everything looks good!